assets/html/themeswap.js

```diff
@@ -38,5 +48,21 @@ function set_theme_from_local_storage() {
       ss.disabled = true;
     });
   }
+  else if(darkTheme !== null) {
+    // If we did find an active theme, we'll (1) add the theme--$(theme) class to <html>
+    document.getElementsByTagName('html')[0].className = "theme--" + darkTheme;
+    // and (2) disable all the other theme stylesheets
+    disabled.forEach(function(ss){
+      if (ss.ownerNode.getAttribute("data-theme-name") !== darkTheme) {
+        ss.disabled = true;
+      }
+    });
+    // Store the theme in localStorage
+    if(typeof(window.localStorage) !== "undefined") {
+      window.localStorage.setItem("documenter-theme", darkTheme);
+    } else {
+      console.error("Browser does not support window.localStorage");
+    }
```

Comment:
If I understand this correctly, this will, if you prefer a dark theme, will then store that dark theme as the preferred theme overall? This would mean that if the user disables the dark-theme on the OS level, docs would still be rendered with the dark theme, right? I think we don't want to do this -- it would be better if the theme would change with the user's OS settings (unless the user has explicitly requested another theme).